lighthouse-core/audits/bootup-time.js

```diff
@@ -124,6 +129,12 @@ class BootupTime extends Audit {
       .filter(result => result.total >= context.options.thresholdInMs)
       .sort((a, b) => b.total - a.total);
 
+
+      if (hadExcessiveChromeExtension) {
+        runWarnings.push('Chrome extensions impacted the performance of this page. ' +
+          'Try running Lighthouse in a clean profile to avoid being impacted by extensions.');
```

Comment:
"clean Chrome profile", maybe? and should we suggest the CLI or not worth it for the average extension/devtools user?

Reply:
done. I think CLI isn't worth it, might just confuse folks :)